src/collector/mod.rs

```diff
@@ -17,6 +17,12 @@ pub use self::multi_collector::MultiCollector;
 mod top_collector;
 pub use self::top_collector::TopCollector;
```

Comment:
Let's keep `TopCollector` crate private. 

In exchange, for backward probabilitiy we can

```rust
#[deprecated]
pub use TopScoreCollector as TopCollector;
```